### PR TITLE
gr-utils: modtool: add option --copyright to add author name

### PR DESCRIPTION
**Pull request by noc0lour**
Maybe this option would be nice to have to add one's name with gr_modtool instead manually or with sed. Since we are using templates anyway we can give an option. Fallback is current behaviour, maybe an interactive behaviour is also desirable?

### CODE REVIEW

**Comment by jmcorgan** [comment]
@mbr0wn Assigned.

File: gr-utils/python/modtool/modtool_add.py [hunk 1]
```diff
@@ -102,8 +103,12 @@ def setup(self, options, args):
             raise ModToolException('Invalid block name.')
         print "Block/code identifier: " + self._info['blockname']
         self._info['fullblockname'] = self._info['modname'] + '_' + self._info['blockname']
+        self._info['copyrightholder'] = options.copyright
+        if self._info['copyrightholder'] is None:
+            self._info['copyrightholder'] = '<+YOU OR YOUR COMPANY+>'
+        elif self._info['is_component']:
+            print "For GNUradio components the FSF is added as copyright holder"
```
Comment (mbr0wn) [inline gr-utils/python/modtool/modtool_add.py]: Please fix this typo (GNUradio -> GNU Radio)

File: gr-utils/python/modtool/modtool_add.py [hunk 2]
```diff
@@ -53,6 +53,8 @@ def setup_parser(self):
                 choices=self._block_types, default=None, help="One of %s." % ', '.join(self._block_types))
         ogroup.add_option("--license-file", type="string", default=None,
                 help="File containing the license header for every source code file.")
+        ogroup.add_option("--copyright", type="string", default=None,
```
Comment (mbr0wn) [inline gr-utils/python/modtool/modtool_add.py]: If `--license-file` is provided, then this option must be ignored. A quick check will help avoiding someone adding both arguments.

**Comment by noc0lour** [comment]
I think I fixed up your comments.

**Comment by mbr0wn** [comment]
:+1:

**Comment by jmcorgan** [comment]
There was a merge conflict on the next branch in templates.py for the copyright change.  I resolved it by using the master branch update; however, this is probably wrong for the mako version of the template.  This will get pushed, but can you check for proper operation and submit an update against next if needed?

**Comment by noc0lour** [comment]
Sure, I can check that. At first glance there is no problem. But I will look into that.

**Pull request merged**